cmd/run.go

```diff
@@ -398,9 +395,18 @@ func runInteractive(ctx context.Context, flags RunFlags) error {
 		workflow.PrintSuccessSummary(ctx)
 	}
 
-	if flags.LaunchStudio || shouldLaunchStudio(ctx, workflow, false) {
-		err = studio.LaunchStudio(ctx, workflow)
+	return maybeLaunchStudio(ctx, workflow, flags)
+}
+
+func maybeLaunchStudio(ctx context.Context, wf *run.Workflow, flags RunFlags) error {
+	canLaunch, numDiagnostics := studio.CanLaunch(ctx, wf)
+	if canLaunch && flags.Watch {
+		return studio.LaunchStudio(ctx, wf)
+	} else if numDiagnostics > 1 {
+		log.From(ctx).PrintfStyled(styles.Info, "\nWe've detected `%d` potential improvements for your SDK.\nThe Speakeasy Studio can fix them for you! Try it out with `speakeasy run --watch`", numDiagnostics)
```

Comment:
```suggestion
		log.From(ctx).PrintfStyled(styles.Info, "\nWe've detected `%d` SDK methods need improvement.\nTo fix automatically run the Speakeasy Studio:", numDiagnostics)
```

Put speakeasy run on a new line

Reply:
Don't love that